app/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers");
+const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require("./errors");
+
+const app = express();
+app.use(express.json());
```

Comment:
We don't technically need this middleware function yet as we only have a get request. Please remove before you merge to main.